core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java

```diff
@@ -390,6 +396,10 @@ private void validateRecordBuildSteps(TypeElement clazz) {
         }
     }
 
+    private Name getPackageName(TypeElement clazz) {
+        return processingEnv.getElementUtils().getPackageOf(clazz).getQualifiedName();
```

Comment:
What is returned if a class is in the default package?

Reply:
It returns `PackageElement` that represents the default package.